.github/CONTRIBUTION.md

```diff
@@ -8,15 +8,24 @@
 
 Run unit tests with:
 
-`yarn test:unit`
+    yarn test:unit
 
 Run integration tests with:
 
-`yarn test:integration`
+    yarn test:integration
 
 Run rendering tests with:
 
-`yarn test:rendering`
+    # Install dependencies
+    yarn --frozen-lockfile
+
+    # Build nebula.js visualization
+    yarn build
+
+    chmod 777 ./test/rendering/commands/run-rendering-test.sh
+    yarn test:local:rendering
```

Comment:
Should this be one copy block?

Reply:
<img width="1058" alt="Screenshot 2022-05-03 at 11 50 08" src="https://user-images.githubusercontent.com/4471489/166434157-1ccc443f-cad0-4183-82ab-0493155c3c5a.png">
it is one copy block. Do I understand you correctly?